CppCoreGuidelines.md

```diff
@@ -13279,7 +13279,7 @@ It also avoids brittle or inefficient workarounds. Convention: That's the way th
         int sz;
     };
 
-    vector<double> v(10);
+    Vector<double> v(10);
```

Comment:
But it didn't mean to reference **that** `vector`, which had its element type fixed.